README.md

```diff
@@ -37,6 +36,29 @@ Both Brotli library and nginx module are under active development.
 
 ## Installation
 
+### Statically compiled
+
+Checkout the latest `ngx_brotli` and build the dependencies:
+
+```
+git clone --recurse-submodules -j8 https://github.com/google/ngx_brotli
+cd ngx_brotli/deps/brotli
+mkdir out && cd out
+cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DCMAKE_C_FLAGS="-Ofast -m64 -march=native -mtune=native -flto -funroll-loops -ffunction-sections -fdata-sections -Wl,--gc-sections" -DCMAKE_CXX_FLAGS="-Ofast -m64 -march=native -mtune=native -flto -funroll-loops -ffunction-sections -fdata-sections -Wl,--gc-sections" -DCMAKE_INSTALL_PREFIX=./installed ..
```

Comment:
`-flto` might break the build on older compilers like in Enterprise Linux 8

Reply:
Not if they use a modern GCC (see devtools / gcc-devtools). You can use a modern compiler without breaking glibc compatibility.